dataduct/utils/hook.py

```diff
@@ -0,0 +1,50 @@
+"""Hook helpers
+"""
+import os
+import imp
+
+
+def default_before_hook(*args, **kwargs):
+    """The default before hook, will act like it's not even there
+    """
+    return args, kwargs
+
+
+def default_after_hook(result):
+    """The default after hook, will act like it's not even there
+    """
+    return result
+
+
+def hook(hook_name):
+    """The hook decorator creator
+
+    """
+    from dataduct.config import Config
+    config = Config()
+    # Try to load the hook. If it fails, just fallback to default hooks.
+    try:
+        hook_directory = os.path.expanduser(config.hooks['path'])
```

Comment:
modify the parse_path function and use that.https://github.com/coursera/dataduct/blob/develop/dataduct/utils/helpers.py